### PR TITLE
Enable usage of porcelain candy dishes

### PR DESCRIPTION
**Pull request by MrFizzyBubbs**
Discussion for this functionality roughly started here: https://discord.com/channels/466605739838930955/756649296774037567/1062567486731603998.

### CODE REVIEW

**Comment by MrFizzyBubbs** [comment]
Alternatively we could only use dishes if you have >=100 in inventory.

File: src/config.ts [hunk 1]
```diff
@@ -40,6 +40,12 @@ You can use multiple options in conjunction, e.g. "garbo nobarf ascend"',
       setting: "",
       help: "Print the current version and exit.",
     }),
+    richbitch: Args.flag({
```
Comment (Pantocyclus) [inline src/config.ts:43]: As Katarn has mentioned in the discord, we should change this to something that isn't as explicit as this. Perhaps `candydish` or `redtooth` or something else descriptive that isn't too lengthy.

Comment (MrFizzyBubbs) [reply src/config.ts:43]: :(

**Review by Pantocyclus — DISMISSED** [review]
I am ok with this, but will leave it to someone else to merge it in if they think it's fine

**Comment by horrible-little-slime** [comment]
can we add a pref for this? It seems like something some people would want to leave on indefinitely

**Pull request merged**